apps/ledger-live-mobile/src/notifications/HookNotifications.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState, useCallback } from "react";
+import { start } from "./braze";
+
+const HookNotifications = () => {
```

Comment:
Why are we naming our hooks like that ? A pretty common convention calls for hook to be prefixed with the "use" keyword. Maybe it's something we can discuss in the team ?

Reply:
Yes totally agree, for now we don't really have a clear convention for those but we can discuss it to put one in place